runtime/near-vm-runner/src/cache.rs

```diff
@@ -102,7 +109,7 @@ pub(crate) fn compile_module_cached_wasmer(
     match cache.get(&(key.0).0) {
         Ok(serialized) => match serialized {
             Some(serialized) => match deserialize_wasmer(serialized.as_slice()) {
-                Ok(module) => Ok(module),
+                Ok(module_or_error) => module_or_error,
                 Err(e) => {
                     error!(target: "runtime", "Cannot deserialize cached contract: {:?}", e);
```

Comment:
It seems that given the current implementation, it is a normal (not an erroneous) operation mode to receive a deserialization error return value.

Reply:
Or not? Hmm, I got confused here